src/Alert/Alert.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React, { type Node } from 'react';
+
+import type { Palette } from '../types';
+import _Alert from './styled';
+
+type Props = {
+  className?: string,
+  children: Node,
+  palette?: Palette
+};
+
+const Alert = ({ className, children, palette, ...props }: Props) => (
+  <_Alert className={className} palette={palette} {...props}>
```

Comment:
Regarding accessibility, it would be great if we could add `role="alert"` as an attribute to `<_Alert>`. See here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role